buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -1,21 +1,22 @@
 object Deps {
-    private const val kotlinVersion = "1.4.21"
+    private const val kotlinVersion = "1.4.30"
```

Comment:
here should be 31 now. also rename to more concrete usage (kotlin version controlled by build.gradle in buildSrc now)